generateConfig.js

```diff
@@ -100,27 +104,15 @@ const generateConfig = ({ intl, env, target, targetFramework = 'react' }) => {
     )
   }
 
-  presets.push(
-    'react',
-    require('./stage1')
-  )
-
   if (targetFramework === 'react') {
-    if (env === 'development') {
-      plugins.push(
-        // Adds component stack to warning messages
-        'transform-react-jsx-source',
-        // Adds __self attribute to JSX which React will use for some warnings
-        'transform-react-jsx-self',
```

Comment:
Это точно больше не нужно?

Reply:
Добавил объяснение в описание к ПР

Reply:
Сходил в https://babeljs.io/docs/en/babel-preset-react, вижу
За флаги development теперь отвечает babel-preset-env? 
Опции `development` и `useBuiltIns` точно дойдут до плагинов?
Выглядит так что их нужно передавать ручками

Reply:
https://github.com/babel/babel/blob/master/packages/babel-preset-react/src/index.js#L14-L15

Reply:
Не понимаю, почему `useBuiltIns: isDev`?)

Reply:
> Will use the native built-in instead of trying to polyfill behavior for any plugins that require one.

ну тип можно без полифила работать, можно и просто убрать, дефолтный false

Reply:
Всегда true ставь и поехали)

Reply:
поставил тру)